.github/workflows/wpt-test.yml

```diff
@@ -1,7 +1,7 @@
 name: Tests
 
 on:
-  pull_request:
+  pull_request_target:
     branches:
```

Comment:
I don’t think you can use this as think `pull_request_target` is against the PR branch not against `main`. Lets delete these two lines and run it for all PR branches.

Reply:
Then I’m at a loss as to why this PR hasn’t triggered this test.

Reply:
I wonder if it has to be merged to take effect?